src/formBuilder/Collapse/Collapse.js

```diff
@@ -5,16 +5,24 @@ import type { Node } from 'react';
 import { createUseStyles } from 'react-jss';
 import { Collapse as RSCollapse } from 'reactstrap';
 import classnames from 'classnames';
+import { faCaretDown, faCaretRight } from '@fortawesome/free-solid-svg-icons';
+import FontAwesomeIcon from '../FontAwesomeIcon';
 
 const useStyles = createUseStyles({
   collapseElement: {
-    '& .disabled': { 'i.toggle-collapse': { cursor: 'default' } },
+    '& .disabled': { '.toggle-collapse': { cursor: 'default' } },
     '& h4': { marginTop: '7px', padding: '13px 10px 10px 10px' },
-    '& i.toggle-collapse': {
+    '& .toggle-collapse': {
       fontSize: '2.3rem',
-      margin: '17px',
       cursor: 'pointer',
-      marginLeft: '0',
+      marginLeft: '33px',
+      '& .fa-caret-right': {
+        marginRight: '9px',
+      },
+    },
+    '& .fa-arrow-up, & .fa-arrow-down': {
```

Comment:
`.fa-arrow-up` corresponds to `faCaretRight`? Otherwise, why define a rule for it? It's not one of the imported fontawesome icons.

Reply:
oh, this is vestigial from something I was trying -- I can just remove.

Reply:
Moved to Card.js, where it actually belongs